### DISCUSSION TITLE
Display Artist or Albumartist on the playback site

### DISCUSSION THREAD

**Discussion by HunLucas** [rern/rAudio#534] [Feature Requests]
Greetings!
If its possible, to display albumartist on the playback screen, if the artist VA or Variousartist, and the albumartists tag isn't empty?
I have a lot of VA albums and I don't see the artist, only VA and the track's title.
But this is always better than tousends of artist on the list page with 1-2songs.
![2021-10-03 (1)](https://user-images.githubusercontent.com/58135504/135754507-1df04880-1a6d-403d-b4d0-47c86dc7bf2c.png)

**Comment by rern** [comment]
An album with tracks by various artists:
- AlbumArtist - VA, Various artists or whatever
- Artist - The artist of each track

Use index bar on the right to navigate long list pages.
